src/components/Navigation/Footer.css

```diff
@@ -0,0 +1 @@
+@import url('https://fonts.googleapis.com/css2?family=Work+Sans&display=swap');
```

Comment:
This isn't specifically related to this pull request, but I think it may be nice to have one Work Sans css file that can be imported where needed or at the top of app.js so that we don't need a css file for each page that needs to import the font. I'm not sure what the most efficient way to do this is though

Reply:
I agree I can make sure to make a consolidated font css file when ppl are finishing their features